client/src/components/leftSidebar/index.tsx

```diff
@@ -7,7 +7,6 @@ import Categorical from "../categorical";
 import * as globals from "../../globals";
 import DynamicScatterplot from "../scatterplot/scatterplot";
 import GeneInfo from "../geneExpression/geneInfo/geneInfo";
-import TopLeftLogoAndTitle from "./topLeftLogoAndTitle";
```

Comment:
Do we need to remove this from `client/src/components/leftSidebar/leftSidebarSkeleton.tsx` as well?

https://github.com/chanzuckerberg/single-cell-explorer/search?q=TopLeftLogoAndTitle&type=